mock/py/mock.py

```diff
@@ -397,6 +399,12 @@ def command_parse():
     return (options, args)
 
 
+def handle_signals(buildroot, number, frame):
+    log.info("\nReceived SIGTERM activating orphansKill")
+    util.orphansKill(buildroot.make_chroot_path())
+    sys.exit(7)
```

Comment:
Exit status 7 isn't documented:
https://github.com/rpm-software-management/mock/blob/master/mock/py/mockbuild/exception.py#L26

Reply:
Good point.  The exit status should be `128 + signum` (to be compatible with shell interpreters).

Reply:
So, if the signum is 13, then exit status should be 141?

Reply:
Since those exit status aren't unused, it sounds good to me.  Btw., is it safe to actually call exit in signal handler?

Reply:
In C run-time `_exit()` and `_Exit()` are async-signal-safe functions (because they do not call `atexit()`-registered callbacks).  No idea about python though...

Reply:
> Exit status 7 isn't documented:
> https://github.com/rpm-software-management/mock/blob/master/mock/py/mockbuild/exception.py#L26

should I document all possible exit codes with signals? like 141, 143, 129

Reply:
in the exceptions file?

Reply:
I would prefer to have them documented symbolically.  For example, this is how `bash(1)` man page documents the codes:

> The return value of a simple command is its exit status, or 128+n if the command is terminated by signal n.